acquire/acquire.py

```diff
@@ -2269,6 +2278,18 @@ def main() -> None:
     log.info("Default Arguments: %s", " ".join(args.config.get("arguments")))
     log.info("")
 
+    # start GUI if requested through CLI / config
+    flavour = "CLI"
+    if args.gui == "always" or (args.gui == "depends" and os.environ.get("KEYSOURCE") == "PROMPT" and sys.argc < 3):
+        flavour = platform.system()
+
+    acquire_gui = gui.GUI(flavour=flavour, upload_available=args.auto_upload)
+    args.output, args.auto_upload, cancel = acquire_gui.wait_for_start(args)
+
+    if cancel:
+        parser.exit(0)
+    # At this stage, in GUI mode, the GUI will be locked, user cannot close window.
```

Comment:
Clicking the close button will show a message "We are currently acquiring this host, please wait". The user cannot close the window (except for using the Task Manager).

Reply:
Can you reword the comment? Something like `From here onwards, the GUI will be locked and cannot be closed because we're acquiring`.

What's the technical reason for this btw? Main thread occupied with running acquire and can't be interrupted I guess?

Reply:
The feature is fine, it's just that the wording could be improved. And I was curious if there was a technical reason :smile: